### PR TITLE
Allow use of Markdown in description and notes

### PR DESCRIPTION
**Pull request by queengooborg**
This PR updates the build/lint scripts to allow the use of Markdown for descriptions and notes internally.  To retain compatibility with consumers, the Markdown is then converted to HTML.  A number of descriptions were updated in this PR to both test and validate that changes to the output are minimal and essential only, and to fix existing issues in our data.

Note: some of our notes already have Markdown formatting.

## Schema Change Proposal

To allow consumers to utilize the Markdown versions if they wish to, I am proposing a change to the schema to convert the `notes` property into an object containing both formats, like so:

```json
{
  notes: {
    md: "**Foo** bar",
    html: "<strong>Foo</strong> bar"
  }
}
```

Alternatively (though I'm less favorable of this), to keep from making a breaking change, we can add a `notesMD` property alongside the HTML version:

```json
{
  notes: "<strong>Foo</strong> bar",
  notesMD: "**Foo** bar"
}
```

## Follow-Up Work

After this PR is approved (but preferably before it is merged), we should go through and convert all of our notes to Markdown format in a bulk update.

### CODE REVIEW

**Comment by teoli2003** [comment]
I'm all for it, though I think this needs a decision from the maintainers.

A question: does this lead to a new major release? 

Maybe we should do it in two phases: changing the schema, allowing some time for consumers (MDN displays these notes, maybe others) to adapt, and then making the textual changes (If I remember well, HTML tags are valid in MD, so it should be ok).

**Comment by queengooborg** [comment]
> A question: does this lead to a new major release?

No, this actually does not change the schema!  To the end user, they'll just see a few notes change, but they're still formatted as HTML.

My apologies, I didn't clarify that the schema proposal is for a _future_ schema change.  This PR purely changes _internal_ formatting for maintainers.  I updated the PR description to clarify this.

**Comment by hamishwillee** [comment]
When's this planned? Very much looking forward to working in markdown.

File: scripts/build/index.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ import fs from 'node:fs/promises';
 import esMain from 'es-main';
 import stringify from 'fast-json-stable-stringify';
 import { compareVersions } from 'compare-versions';
+import { marked } from 'marked';
```
Comment (ddbeck) [inline scripts/build/index.ts:9]: I'd strongly discourage you from doing this specific thing. Marked doesn't fully implement CommonMark. If you ever try to do something else with Markdown or consume the Markdown using another tool (e.g., an off the shelf linter), you'll be very sad. Consider using something like [remark and remark-rehype](https://remark.js.org/) instead.

Comment (queengooborg) [reply scripts/build/index.ts:9]: Do you happen to know what parts of CommonMark this library doesn't implement?

Comment (ddbeck) [reply scripts/build/index.ts:9]: > Do you happen to know how its output differs from the spec?

Not specifically. The [marked docs](https://marked.js.org/#specifications) link to a discussion that was last updated in 2022, but several areas (including links) don't pass the CommonMark tests, but it doesn't show specifically which tests were failing.

> it may not be something to worry about

This is how it _always_ starts. I've been on no less than three projects where somebody said, "we're not using this for anything big, it'll be no problem to use a non-standard Markdown implementation" and before long someone finds another use for the non-standard Markdown parser and then, finally, the day arrives where it becomes a problem that requires substantial rework to fix. Please don't let me be the Cassandra of Markdown.

It's so much easier to do the right thing when it's not a big deal than it is to retrofit an existing body of work.

Comment (queengooborg) [reply scripts/build/index.ts:9]: I cloned the repository locally and made some modifications to determine what exactly is failing in the tests for `marked`.  I have [uploaded a file containing the (simplified) output log of the failing tests](https://github.com/user-attachments/files/16344340/marked-failing-commonmark-tests.txt).

To summarize the results, here's what I've found:

Examples 32, 33 and 503 are about encoding characters with accents a specific way within URLs (`f&ouml;&ouml;` (`föö`) becomes `f%C3%B6%C3%B6`).  It appears that `marked` decodes characters with accents, causing these examples to fail tests.  However, since browsers now automatically encode URLs, I believe this is a non-issue.

Example 507 has to do with the specific white spacing needed to parse links with titles.  The CommonMark spec requires only spaces, tabs or single line endings between the link and any title text, otherwise it should all be included as a part of the URL.  `marked` treats a non-breaking space as valid whitespace.

Examples 497, 512, 518-520, 524, 526, 528, 532-533, 536 and 538 are basically how the parser should handle syntax errors related to links (such as overlapping syntax usage, like `[foo [bar](/uri)](/uri)`).

Example 540 has to do with encoding/decoding non-ASCII characters in reference links (`ß` <-> `SS`).  For BCD's use case, this is a non-issue, as we do not use reference links anywhere in BCD (and would not be able to in notes/descriptions anyways).

Examples 573-577, 585 and 589 are all about stripping out Markdown syntax from the alt text in images.  For BCD, this is again a non-issue, as images are not permitted in notes/descriptions.

Comment (ddbeck) [reply scripts/build/index.ts:9]: For your consideration https://github.com/markdown-it/markdown-it  (via [Eleventy](https://www.11ty.dev/docs/languages/markdown/))

**Review by caugner — APPROVED** [review]
I'm in favor of using marked here, because it is the lightest markdown parser.

Tested in a fresh directory (`npm init -y`):
- `npm add marked && du -h .` -> 972K
- `npm add markdown-it && du -h .` -> 1.8M
- `npm add remark remark-rehype && du -h .` -> 4.1M

File: scripts/build/index.ts (hunk 1 above)
Comment (caugner) [reply scripts/build/index.ts:9]: Let's be pragmatic and choose `marked`. We can reconsider when we actually have an issue, but afaik for now this will be used almost exclusively for links, bold, and italic.

File: scripts/build/index.ts [hunk 2]
```diff
@@ -109,6 +122,44 @@ export const addVersionLast = (feature: WalkOutput): void => {
   }
 };
 
+/**
+ * Convert descriptions and notes from Markdown to HTML
+ * @param {WalkOutput} feature The BCD to perform mirroring on
```
Comment (caugner) [inline scripts/build/index.ts:127]: This description seems wrong, as we're transforming, not mirroring.

**Review by caugner — CHANGES_REQUESTED** [review]
I ran an`npm run build` diff, and here's what I got (`git diff --minimal`):

https://gist.github.com/caugner/51e20d9a8a0e7ddce5c6e41073518105

This revealed a couple of issues:

| Before | After |
| ------ | ----- | 
| `<code>https://z.com/x?a=true&amp;b=false</code>` | `<code><a href=\"https://z.com/x?a=true&amp;b=false\">https://z.com/x?a=true&amp;b=false</a></code>`
| `<code>&quot;negotiate&quot;</code>` | `<code>\"negotiate\"</code>` |
| `Hyphenation dictionary for Norwegian (Bokmål) (no, no-*, nb, nb-*)` | `Hyphenation dictionary for Norwegian (Bokmål) (no, no-<em>, nb, nb-</em>)` |
| `Namespaces (<code>*\|*</code>)` | `Namespaces (<code><em>\|</em></code>)` |
| `<code>calc(1*2*3)</code>` | `<code>calc(1<em>2</em>3)</code>` |
| `<code>Accept: */*</code>` | `<code>Accept: <em>/</em></code>` |
| `<code>__defineGetter__</code>` | `<code><strong>defineGetter</strong></code>` |
| `'http://127.0.0.1'` | `'<a href=\"http://127.0.0.1\">http://127.0.0.1</a>'` |

The bottom line is we can probably only land this if we also
1. convert all `<code>`s to backticks (and forbid `<code>` tags internally), and
2. escape most asterisks.

**Comment by queengooborg** [comment]
We'll definitely want to do those bits about `<code>` blocks and asterisk escaping -- that will be performed in a migration PR that will be performed right after this PR is merged!

**Comment by queengooborg** [comment]
I have just pushed a few commits that resolve issues in the Markdown conversion.  Looking at the diff, I noticed that a good chunk of differences could be resolved programmatically, and then the remaining issues were a small handful that were either issues in the markup that need to be resolved in the main branch anyways, or small tweaks that can be done within this PR.

I pushed a commit that performs the small handful of fixes, and cherry-picked the ones that are not exclusively for Markdown conversion and opened #24911.  The remainder of the diff is now all due to notes that are already Markdown-formatted.

**Review by caugner — APPROVED** [review]
LGTM, except one nit.

Compared this branch's build against main with [jd](https://github.com/josephburnett/jd):

```
jd --color data_ma*.json | sed 's/\\u003c/</g' | sed 's/\\u003e/>/g' | sed 's/\\u0026/\&/g'`)
```

**Comment by caugner** [comment]
The diff looks good now, so I'm merging.

FWIW Here's the full diff:

<details><summary><code>jd --color data_main.json build/data.json | sed 's/\\u003c/</g' | sed 's/\\u003e/>/g' | sed 's/\\u0026/\&/g'`)</code></summary>

```
    ✭
@ ["__meta","timestamp"]
- "2024-11-15T12:43:01.193Z"
+ "2024-11-15T13:23:49.247Z"
@ ["api","CredentialsContainer","create","publicKey_option","attestation","direct","__compat","support","firefox_android","notes"]
- "<code>credentialCreationData.attestationConveyancePreferenceOption</code> value \"direct\" is not supported. See https://bugzil.la/1550164"
+ "<code>credentialCreationData.attestationConveyancePreferenceOption</code> value \"direct\" is not supported. See <a href=\"https://bugzil.la/1550164\">https://bugzil.la/1550164</a>"
@ ["api","CredentialsContainer","create","publicKey_option","attestation","enterprise","__compat","support","firefox_android","notes"]
- "<code>credentialCreationData.attestationConveyancePreferenceOption</code> value \"enterprise\" is not supported. See https://bugzil.la/1550164"
+ "<code>credentialCreationData.attestationConveyancePreferenceOption</code> value \"enterprise\" is not supported. See <a href=\"https://bugzil.la/1550164\">https://bugzil.la/1550164</a>"
@ ["api","HTMLInputElement","selectionchange_event","__compat","support","chrome","notes"]
- "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href='https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event'><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;input&gt;</code> elements."
+ "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href=\"https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event\"><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;input&gt;</code> elements."
@ ["api","HTMLInputElement","selectionchange_event","__compat","support","chrome_android","notes"]
- "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href='https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event'><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;input&gt;</code> elements."
+ "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href=\"https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event\"><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;input&gt;</code> elements."
@ ["api","HTMLInputElement","selectionchange_event","__compat","support","edge","notes"]
- "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href='https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event'><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;input&gt;</code> elements."
+ "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href=\"https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event\"><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;input&gt;</code> elements."
@ ["api","HTMLInputElement","selectionchange_event","__compat","support","oculus","notes"]
- "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href='https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event'><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;input&gt;</code> elements."
+ "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href=\"https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event\"><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;input&gt;</code> elements."
@ ["api","HTMLInputElement","selectionchange_event","__compat","support","opera","notes"]
- "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href='https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event'><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;input&gt;</code> elements."
+ "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href=\"https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event\"><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;input&gt;</code> elements."
@ ["api","HTMLInputElement","selectionchange_event","__compat","support","opera_android","notes"]
- "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href='https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event'><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;input&gt;</code> elements."
+ "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href=\"https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event\"><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;input&gt;</code> elements."
@ ["api","HTMLInputElement","selectionchange_event","__compat","support","samsunginternet_android","notes"]
- "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href='https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event'><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;input&gt;</code> elements."
+ "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href=\"https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event\"><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;input&gt;</code> elements."
@ ["api","HTMLInputElement","selectionchange_event","__compat","support","webview_android","notes"]
- "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href='https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event'><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;input&gt;</code> elements."
+ "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href=\"https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event\"><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;input&gt;</code> elements."
@ ["api","HTMLTextAreaElement","selectionchange_event","__compat","support","chrome","notes"]
- "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href='https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event'><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;textarea&gt;</code> elements."
+ "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href=\"https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event\"><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;textarea&gt;</code> elements."
@ ["api","HTMLTextAreaElement","selectionchange_event","__compat","support","chrome_android","notes"]
- "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href='https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event'><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;textarea&gt;</code> elements."
+ "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href=\"https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event\"><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;textarea&gt;</code> elements."
@ ["api","HTMLTextAreaElement","selectionchange_event","__compat","support","edge","notes"]
- "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href='https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event'><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;textarea&gt;</code> elements."
+ "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href=\"https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event\"><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;textarea&gt;</code> elements."
@ ["api","HTMLTextAreaElement","selectionchange_event","__compat","support","oculus","notes"]
- "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href='https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event'><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;textarea&gt;</code> elements."
+ "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href=\"https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event\"><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;textarea&gt;</code> elements."
@ ["api","HTMLTextAreaElement","selectionchange_event","__compat","support","opera","notes"]
- "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href='https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event'><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;textarea&gt;</code> elements."
+ "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href=\"https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event\"><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;textarea&gt;</code> elements."
@ ["api","HTMLTextAreaElement","selectionchange_event","__compat","support","opera_android","notes"]
- "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href='https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event'><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;textarea&gt;</code> elements."
+ "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href=\"https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event\"><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;textarea&gt;</code> elements."
@ ["api","HTMLTextAreaElement","selectionchange_event","__compat","support","samsunginternet_android","notes"]
- "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href='https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event'><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;textarea&gt;</code> elements."
+ "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href=\"https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event\"><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;textarea&gt;</code> elements."
@ ["api","HTMLTextAreaElement","selectionchange_event","__compat","support","webview_android","notes"]
- "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href='https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event'><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;textarea&gt;</code> elements."
+ "A <code>selectionchange</code> event is fired on <code>Document</code>, see <a href=\"https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event\"><code>Document</code>'s <code>selectionchange</code> event</a>. See <a href='https://crbug.com/40840956'>bug 40840956</a> for firing the event on <code>&lt;textarea&gt;</code> elements."
@ ["api","Permissions","permission_notifications","__compat","support","firefox","notes"]
- "Alias for the `push` permission (returns the same value)."
+ "Alias for the <code>push</code> permission (returns the same value)."
@ ["api","Permissions","permission_notifications","__compat","support","firefox_android","notes"]
- "Alias for the `push` permission (returns the same value)."
+ "Alias for the <code>push</code> permission (returns the same value)."
@ ["api","Permissions","permission_push","__compat","support","firefox","notes"]
- "Alias for the `notifications` permission (returns the same value)."
+ "Alias for the <code>notifications</code> permission (returns the same value)."
@ ["api","Permissions","permission_push","__compat","support","firefox_android","notes"]
- "Alias for the `notifications` permission (returns the same value)."
+ "Alias for the <code>notifications</code> permission (returns the same value)."
@ ["css","properties","color-interpolation","__compat","support","chrome","notes"]
- "Only the default value of `sRGB` is implemented"
+ "Only the default value of <code>sRGB</code> is implemented"
@ ["css","properties","color-interpolation","__compat","support","chrome_android","notes"]
- "Only the default value of `sRGB` is implemented"
+ "Only the default value of <code>sRGB</code> is implemented"
@ ["css","properties","color-interpolation","__compat","support","edge","notes"]
- "Only the default value of `sRGB` is implemented"
+ "Only the default value of <code>sRGB</code> is implemented"
@ ["css","properties","color-interpolation","__compat","support","oculus","notes"]
- "Only the default value of `sRGB` is implemented"
+ "Only the default value of <code>sRGB</code> is implemented"
@ ["css","properties","color-interpolation","__compat","support","opera","notes"]
- "Only the default value of `sRGB` is implemented"
+ "Only the default value of <code>sRGB</code> is implemented"
@ ["css","properties","color-interpolation","__compat","support","opera_android","notes"]
- "Only the default value of `sRGB` is implemented"
+ "Only the default value of <code>sRGB</code> is implemented"
@ ["css","properties","color-interpolation","__compat","support","safari","notes"]
- "Only the default value of `sRGB` is implemented"
+ "Only the default value of <code>sRGB</code> is implemented"
@ ["css","properties","color-interpolation","__compat","support","safari_ios","notes"]
- "Only the default value of `sRGB` is implemented"
+ "Only the default value of <code>sRGB</code> is implemented"
@ ["css","properties","color-interpolation","__compat","support","samsunginternet_android","notes"]
- "Only the default value of `sRGB` is implemented"
+ "Only the default value of <code>sRGB</code> is implemented"
@ ["css","properties","color-interpolation","__compat","support","webview_android","notes"]
- "Only the default value of `sRGB` is implemented"
+ "Only the default value of <code>sRGB</code> is implemented"
@ ["css","properties","color-interpolation","__compat","support","webview_ios","notes"]
- "Only the default value of `sRGB` is implemented"
+ "Only the default value of <code>sRGB</code> is implemented"
@ ["css","properties","position-area","__compat","support","firefox","notes"]
- "The property is parsed and accepted, but it has no effect yet, because both <a href='https://developer.mozilla.org/docs/Web/CSS/anchor-name'>`anchor-name`</a> and <a href='https://developer.mozilla.org/docs/Web/CSS/position-anchor'>`position-anchor`</a> are not yet supported."
+ "The property is parsed and accepted, but it has no effect yet, because both <a href='https://developer.mozilla.org/docs/Web/CSS/anchor-name'><code>anchor-name</code></a> and <a href='https://developer.mozilla.org/docs/Web/CSS/position-anchor'><code>position-anchor</code></a> are not yet supported."
@ ["css","properties","position-area","__compat","support","firefox_android","notes"]
- "The property is parsed and accepted, but it has no effect yet, because both <a href='https://developer.mozilla.org/docs/Web/CSS/anchor-name'>`anchor-name`</a> and <a href='https://developer.mozilla.org/docs/Web/CSS/position-anchor'>`position-anchor`</a> are not yet supported."
+ "The property is parsed and accepted, but it has no effect yet, because both <a href='https://developer.mozilla.org/docs/Web/CSS/anchor-name'><code>anchor-name</code></a> and <a href='https://developer.mozilla.org/docs/Web/CSS/position-anchor'><code>position-anchor</code></a> are not yet supported."
@ ["css","properties","scroll-timeline-axis","__compat","support","firefox","notes",2]
- "The `@scroll-timeline` at-rule is replaced with the longhand properties `scroll-timeline-name` and `scroll-timeline-axis` and the shorthand property `scroll-timeline`."
+ "The <code>@scroll-timeline</code> at-rule is replaced with the longhand properties <code>scroll-timeline-name</code> and <code>scroll-timeline-axis</code> and the shorthand property <code>scroll-timeline</code>."
@ ["css","properties","scroll-timeline-name","__compat","support","firefox","notes",1]
- "The `@scroll-timeline` at-rule is replaced with the longhand properties `scroll-timeline-name` and `scroll-timeline-axis` and the shorthand property `scroll-timeline`."
+ "The <code>@scroll-timeline</code> at-rule is replaced with the longhand properties <code>scroll-timeline-name</code> and <code>scroll-timeline-axis</code> and the shorthand property <code>scroll-timeline</code>."
@ ["javascript","operators","import","__compat","support","deno","notes"]
- "Bundled Deno applications (using `deno compile`) do not support dynamic imports"
+ "Bundled Deno applications (using <code>deno compile</code>) do not support dynamic imports"
@ ["svg","global_attributes","color-interpolation","__compat","support","chrome","notes"]
- "Only the default value of `sRGB` is implemented"
+ "Only the default value of <code>sRGB</code> is implemented"
@ ["svg","global_attributes","color-interpolation","__compat","support","chrome_android","notes"]
- "Only the default value of `sRGB` is implemented"
+ "Only the default value of <code>sRGB</code> is implemented"
@ ["svg","global_attributes","color-interpolation","__compat","support","edge","notes"]
- "Only the default value of `sRGB` is implemented"
+ "Only the default value of <code>sRGB</code> is implemented"
@ ["svg","global_attributes","color-interpolation","__compat","support","oculus","notes"]
- "Only the default value of `sRGB` is implemented"
+ "Only the default value of <code>sRGB</code> is implemented"
@ ["svg","global_attributes","color-interpolation","__compat","support","opera","notes"]
- "Only the default value of `sRGB` is implemented"
+ "Only the default value of <code>sRGB</code> is implemented"
@ ["svg","global_attributes","color-interpolation","__compat","support","opera_android","notes"]
- "Only the default value of `sRGB` is implemented"
+ "Only the default value of <code>sRGB</code> is implemented"
@ ["svg","global_attributes","color-interpolation","__compat","support","safari","notes"]
- "Only the default value of `sRGB` is implemented"
+ "Only the default value of <code>sRGB</code> is implemented"
@ ["svg","global_attributes","color-interpolation","__compat","support","safari_ios","notes"]
- "Only the default value of `sRGB` is implemented"
+ "Only the default value of <code>sRGB</code> is implemented"
@ ["svg","global_attributes","color-interpolation","__compat","support","samsunginternet_android","notes"]
- "Only the default value of `sRGB` is implemented"
+ "Only the default value of <code>sRGB</code> is implemented"
@ ["svg","global_attributes","color-interpolation","__compat","support","webview_android","notes"]
- "Only the default value of `sRGB` is implemented"
+ "Only the default value of <code>sRGB</code> is implemented"
@ ["svg","global_attributes","color-interpolation","__compat","support","webview_ios","notes"]
- "Only the default value of `sRGB` is implemented"
+ "Only the default value of <code>sRGB</code> is implemented"
```

</details>
